packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6555,4 +6555,22 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return Promise.reject(new Error('Error sending reaction, service url not found.'));
   }
+
+  /**
+   * Method to enable or disable reactions inside the meeting.
+   *
+   * @param  {boolean} enable - enable or disable reactions
+   * @returns {Promise}
+   * @public
+   * @memberof Meeting
```

Comment:
Please do not define the the parameter types in JSDoc in TS files, they already defined in the function definition. Also the `@public` and `@memberof` tags because you are using the `public` keyword and TS know `toggleReactions` is member of the `Meeting` class

Reply:
I'm not sure the dos are generate using typescript yet. I noticed typescript is not even built at the moment.

Reply:
same as above 

@maxinteger  do you have cycles to help us with the migration ?